packages/core/src/browser/tree/tree-model.ts

```diff
@@ -228,6 +237,7 @@ export class TreeModel implements ITreeModel, SelectionProvider<Readonly<ISelect
         if (node) {
             this.doOpenNode(node);
         }
+        this.onOpenNodeEmitter.fire(raw);
```

Comment:
It should be `this.onOpenNodeEmitter.fire(node)`

Should it happen only when `node` is not undefined?

Reply:
see above